client/public/index.html

```diff
@@ -2,7 +2,9 @@
 <html>
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" /><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dosis:wght@500&family=Nunito+Sans&display=swap" rel="stylesheet">
```

Comment:
google fonts should be imported into the SCSS file with other stylings

@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');

but replace the url address with the ones for your fonts